compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/idx-method-no-outlining-wildcard.js

```diff
@@ -0,0 +1,23 @@
+// @customMacros(idx.*.b)
```

Comment:
can you add a test case where the wildcard is at the end? i'm assuming that `idx.a.*` would match both `idx.a.b` and `idx.a.c.d` (?)